app/assets/stylesheets/application.scss

```diff
@@ -282,23 +290,23 @@ table {
   }
 
   .sorting {
-    background: image-url('sort_both.png') no-repeat center right;
+    background: url('sort_both.png') no-repeat center right;
```

Comment:
maybe to make sure this isn't missed as easily and reduce the size of app.scss, extract these classes to data-tables.scss